### DISCUSSION TITLE
礼貌提问 ， 多久开始适配3.12呢

### DISCUSSION THREAD

**Discussion by irx999** [Antonoko/Windrecorder#163] [General 闲谈区]
![image](https://github.com/yuka-friends/Windrecorder/assets/72070657/0973872e-85de-46fa-83c5-979b059db768)

**Comment by Antonoko** [comment]
（理论上逻辑部分代码应该都是没问题的，不支持 3.12 主要问题应该在所需依赖还未更新支持？……需要一些调查✍（俺对偏底层的运作知之甚少，苦手了
